application-analytics-ui/src/main/resources/Analytics/Code/Macros/RowEvolutionJSON.xml

```diff
@@ -41,11 +41,14 @@
   'period' : $request.period,
   'date' : $request.date,
   'module' : 'API',
-  'method' : 'Actions.getPageUrl',
   'format' : 'json'
 })
 #if ($request.macroName == 'MostViewedPages')
   #set ($discard = $parameters.put('pageUrl', $request.parameter))
```

Comment:
You'll have probably some conflicts over the `request.parameter` that will be `request.rowIdentifier`